repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -146,6 +147,29 @@ export const getLinesList = (
     return linesList;
 };
 
+export const getGroupOfLinesList = (
+    operatorIdentifier: string,
+    userPeriodTicket: PeriodMultipleServicesTicket | MultiOperatorMultipleServicesTicket | SchemeOperatorFlatFareTicket,
+    lines: Line[],
+): GroupOfLines[] => {
+    const lineReferences = lines.map(line => line.id);
+    return [
+        {
+            version: '1.0',
+            id: `${operatorIdentifier}@groupOfLines@1`,
+            Name: {
+                $t: `A group of available ${isHybridTicket(userPeriodTicket) ? 'additional' : ''} services.`,
```

Comment:
Minor; you could put the extra space after additional inside the ternary so you don get a double space for non-hybrid